### ISSUE TITLE
Render to Android

### ISSUE THREAD

**Issue by ocrickard** [ocrickard/Theodolite#19]
Ideally, we'd be able to compile Theodolite to both iOS and Android view systems. We may not be able to reuse all the code we have, but much of it should be able to map well.

**Comment by ocrickard** [comment]
@amonshiz are you looking at this?

**Comment by amonshiz** [comment]
No, I have not looked at this yet. I did start looking into what the state of the art was for building Swift for Android, and everything seems pretty old. I will check in with @modocache to see if he knows more.

**Comment by ocrickard** [comment]
OK, great! Let me know what we need to do here.
